src/components/AddBookForm.js

```diff
@@ -1,11 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { bookAdded } from '../redux/booksSlice';
 
-const AddBookForm = () => (
-  <form>
-    <input name="title" placeholder="Book title" />
-    <input name="author" placeholder="Author" />
-    <button type="button" onClick={null}>ADD BOOK</button>
-  </form>
-);
+const AddBookForm = () => {
+  const dispatch = useDispatch();
 
+  const initialData = {
+    title: '', author: '', category: 'Economy',
+  };
+
+  const [data, setData] = useState(initialData);
+
+  const handleOnChange = (e) => {
+    setData({ ...data, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(bookAdded(data));
+  };
```

Comment:
- [ ] [_OPTIONAL_] Great job so far, after submitting a form the data still persists on the form, kindly fix that.

i.e `e.target.reset();`